packages/tasks/src/model-libraries.ts

```diff
@@ -304,6 +304,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: false,
 		countDownloads: `path:"model.safetensors"`,
 	},
+	"mlc-llm": {
+		prettyLabel: "MLC-LLM",
+		repoName: "MLC-LLM",
+		repoUrl: "https://github.com/mlc-ai/mlc-llm",
+		filter: true,
```

Comment:
```suggestion
		filter: false,
```

`filter: true` would add mlc-llm library to the libraries list on the left side on hf.co/models. Given it's quite small for now ([3 models](https://huggingface.co/models?other=mlc-llm)?), let's have it hidden.
![image](https://github.com/huggingface/huggingface.js/assets/11801849/3cfd82b5-f4b7-43da-aeef-eb213f5b6ed1)

Reply:
Sorry, I didn't see https://github.com/huggingface/huggingface.js/pull/788#discussion_r1668737141 before reviewing

Reply:
ref: https://github.com/huggingface/huggingface.js/pull/788#discussion_r1668737141

Reply:
Let's do `filter: false` to begin with and then update this as the changes trickle down.